lib/pocketape.dart

```diff
@@ -23,15 +25,20 @@ abstract class Pocketape {
           onDone: controller.close,
         );
         if (_count == 1) {
+          print("Start");
+          if (Platform.isAndroid && !(await _requestPermission())) {
+            controller.addError(CameraPermissionDeniedException("Camera permission denied"));
```

Comment:
Minor: as a message use: "Camera permission is required on Android in order to use ARCore."